src/platform/visualc/config.h

```diff
@@ -39,7 +39,7 @@
 /* Enable memory function inlining in */
 #define C_CORE_INLINE 1
 
-/* Define to 1 to enable FluidSynth MIDI synthesiser */
+/* Define to 1 to enable FluidSynth MIDI synthesizer */
```

Comment:
It was British spelling ;)

Reply:
I'm in Canada (use British spelling here too vs American), but haven't seen 'ise'  spelling in general.. always 'ize'  :-)  I'm not sure how to get google hit counts anymore - can we go with the most common (greater hit count?).

Reply:
Let the numbers decide?
 - **sizer** has 66.8 M matches
 - **siser** has 1.8 M matches

![2020-12-17_08-35](https://user-images.githubusercontent.com/1557255/102515803-096c2180-4043-11eb-9c0b-d04056cc4dad.png)
Turns out mobile doesn't show counts. 
![2020-12-17_08-36](https://user-images.githubusercontent.com/1557255/102515795-07a25e00-4043-11eb-89ce-11d22fa46945.png)